.github/workflows/release.yaml

```diff
@@ -289,6 +289,36 @@ jobs:
           asset_content_type: application/octet-stream
         if: ${{ env.DRY_RUN != 'true' }}
 
+      - name: Generate SBOM (spdx)
```

Comment:
Looks like there's a public spdx-sbom-generator image (spdx/spdx-sbom-generator). Could you use that rather than downloading the binary?

Reply:
I tried that at first but that image wasn't built to be used as base. It just expose the spdx-sbom-generator command. I would have to hack the entry-point to make it work as a container in github-action. Using the binary is more flexible and it doesn't add docker dependency which are 2 advantages imo.